Edit

duyhl512 opened the pull request.

duyhl512 committed:
git commit  -m  "Added test case for the new feature"
[ master  1b3b4b9] Added test case for the new feature
 1  file changed,  1  insertion(+),  1  deletion(-)

duyhl512 committed:
git commit

duyhl512 committed:
git commit -m "Added TestPlanDAO and TestPlan DTO"

duyhl512 commented:
Add

duyhl512 committed:
git commit -m "Added TestRun and Milestone DTOs but write nothing"
git commit  -m "modified TestCaseHandler"

duyhl512 commented:
Re - moddified TestCase Handler

thang1160 commented on src/main/java/com/example/starter/dao/TestPlanDAO.java:
```diff
@@ -0,0 +1,60 @@
+
+        try {
+            conn = getConnection();
+            ps = conn.prepareStatement("exec [GetTestPlansByProjectId] ?;");
```
Thiếu file sql của GetTestPlansByProjectId

thang1160 commented on src/main/java/com/example/starter/dao/TestPlanDAO.java:
```diff
@@ -0,0 +1,60 @@
+                    + "VALUES (?, ?, ?, ?, ?)");
+        } catch (Exception e) {
+            _LOGGER.log(Level.SEVERE, "add testplan to database failed", e);
+        }
```
chưa đóng connection và PreparedStatement

duyhl512 committed:
git add database/GetTestPlansByProjectId.sql
git commit -m "GetTestPlansByProjectId.sql"

thang1160 commented on src/main/java/com/example/starter/dto/TestCase.java:
```diff
@@ -2,36 +2,36 @@
-    private Integer userId;
     private Integer statusId;
     private boolean deleted;
+    private Integer createOn;
```
Sao cái này lại là Integer nhỉ

thang1160 commented on src/main/java/com/example/starter/dto/TestCase.java:
```diff
@@ -2,36 +2,36 @@
-        this.userId = userId;
-        this.statusId = 1;
-        this.deleted = false;
-        this.updatedBy = userId;
```
statusId = 1, deleted = false và updatedBy = userId là default value đã chốt trong cuộc họp chủ nhật rồi

thang1160 commented on src/main/java/com/example/starter/handler/TestCaseHandler.java:
```diff
@@ -18,17 +18,21 @@ public static void addTestCase(RoutingContext rc) {
                 String expectedResult = json.getString("expected-result");
-                Integer userId = json.getInteger("user-id");
+                Integer status_id = json.getInteger("status-id");
+                Boolean isDeleted = json.getBoolean("is-deleted");
```
Tương tự những params này không cần thiết lấy ra, em check lại swagger trước khi sửa lại

duyhl512 committed:
git commit -m "modified TestPlanDAO.java, TestCase.java, TestCaseHandler.java"

duyhl512 commented:
Em đã chỉnh sửa lại các file trên, anh xem xét lại giúp e

duyhl512 committed:
git commit  -m  "Added test case handler"
[ master  1b3b9b9] Added test case handler
 2  files changed,  2  insertions(+),  2  deletions(-)
